docs/modules/datasources/pages/entity_datasource.adoc

```diff
@@ -26,7 +26,28 @@ The default if the `query` field is empty is `alarms()` (return a list of alarm
 
 === alarms([attribute])
 
-The `alarms()` function will query the OpenNMS alarms ReST API.
+The `alarms()` function queries the OpenNMS alarms REST API.
+
+=== locations()
+
+The `locations()` query returns a list of all available node locations. 
+Use this variable with `nodeFilter()` to reduce the number of nodes displayed by location.
+
+Examples: 
+```
+nodeFilter(location='Default')
+nodeFilter(location='$location')
+```
```

Comment:
The triple backtick notation for code blocks isn't used by the documentation system.  This should be a list instead.
```suggestion
.Example variable expression
* nodeFilter(location='Default')
* nodeFilter(location='$location')
```